core/src/main/java/nl/nn/adapterframework/core/PipeLineExits.java

```diff
@@ -20,6 +20,9 @@
 
 import lombok.Getter;
 
+/**
+ * If no exits are specified, a default one is created with path="READY" and state="SUCCESS".
+ */
```

Comment:
Het zou helemaal tof zijn als dit element ook echt een beschrijvend stuk informatie laat zien. Bijvoorbeeld:

Pipeline exit container in which all (required) {@link PipeLineExit Exit}s must be defined. Multiple exits may be provided each with their unique name (once #2877 is merged!).
```
   <Exits>
      <Exit path="READY" state="SUCCESS" />
      <Exit path="Created" state="ERROR" code="201" empty="true" />
      <Exit path="NotModified" state="ERROR" code="304" empty="true" />
      <Exit path="BadRequest" state="ERROR" code="400" empty="true" />
      <Exit path="NotAuthorized" state="ERROR" code="401" empty="true" />
      <Exit path="NotAllowed" state="ERROR" code="403" empty="true" />
      <Exit path="Teapot" state="SUCCESS" code="418" />
      <Exit path="ServerError" state="ERROR" code="500" />
   </Exits>
```

Reply:
Jaco is already editing PipeLineExit in the PR you referenced. I cannot follow your comments right now.

Reply:
I checked that Jaco's PR has been merged. I will keep in touch with him and continue with this PR.

Reply:
Is het beschrijvende stuk tekst nog steeds nodig? Jaco heeft er al voorbeelden bij gezet van Exit-s.

Reply:
Voeg alsjeblieft een beschrijvend stuk informatie toe. Bijvoorbeeld:

Pipeline exit container in which all (required) {@link PipeLineExit Exit}s must be defined. Multiple exits may be provided each with their unique name (once #2877 is merged!).

Ook vind ik het voorbeeld hier meer op zijn plaats.
```
   <Exits>
      <Exit path="READY" state="SUCCESS" />
      <Exit path="Created" state="ERROR" code="201" empty="true" />
      <Exit path="NotModified" state="ERROR" code="304" empty="true" />
      <Exit path="BadRequest" state="ERROR" code="400" empty="true" />
      <Exit path="NotAuthorized" state="ERROR" code="401" empty="true" />
      <Exit path="NotAllowed" state="ERROR" code="403" empty="true" />
      <Exit path="Teapot" state="SUCCESS" code="418" />
      <Exit path="ServerError" state="ERROR" code="500" />
   </Exits>
```

Reply:
Gedaan. Het voorbeeld van Jaco staat hier nu ook.